### ISSUE TITLE
APA WG comment: Requested Additional WCAG 2.1 References

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/imsc#520]
In addition to the guidelines and success criteria you already reference, we would like to see references to the following WCAG 2.1 success criteria that are also applicable to the TTML profiles specification:

1. [Success Criterion 1.4.3 Contrast (Minimum)](https://www.w3.org/TR/2018/REC-WCAG21-20180605/#contrast-minimum) (applying to the text-only profile)
2. [Success Criterion 1.4.11 Non-text Contrast](https://www.w3.org/TR/2018/REC-WCAG21-20180605/#non-text-contrast) (applying to the image-only profile)
3. [Success Criterion 3.1.1 Language of Page](https://www.w3.org/TR/2018/REC-WCAG21-20180605/#language-of-page) (using the xml:lang attribute)
4. [Success Criterion 3.1.2 Language of Parts](https://www.w3.org/TR/2018/REC-WCAG21-20180605/#language-of-parts) (using the xml:lang attribute)

**Comment by palemieux** [comment]
@michael-n-cooper Success Criterion 1.4.11 probably does not apply to IMSC since images are images of text, which fall under 1.4.3.

**Comment by gzimmermann** [comment]
> @michael-n-cooper Success Criterion 1.4.11 probably does not apply to IMSC since images are images of text, which fall under 1.4.3.

So, in the image profile, are images only included as images of text? I thought that an author could also include an image showing an emoticon, for example.

**Comment by palemieux** [comment]
>I thought that an author could also include an image showing an emoticon, for example.

@gzimmermann I am surprised that emoticons are included in 1.4.11. Where is the line drawn between emoticons and ideograms, e.g. Japanese Kanji?

**Comment by gzimmermann** [comment]
> @gzimmermann I am surprised that emoticons are included in 1.4.11. Where is the line drawn between emoticons and ideograms, e.g. Japanese Kanji? In other words, why should emoticons not be treated like text and be subject to Success Criterion 1.4.6?

For authors of TTML, the difference between 1.4.3 AA Contrast (minimum) and 1.4.11 AA Non-text contrast is the minimum contrast ratio (4.5 for 1.4.3, 3 for 1.4.11).  Although emoticons are not mentioned in the explanations ("understanding") of either success criterion, i consider an emoticon as a symbol which is basically easier to recognize than a character because it is not so fine-grained. Cf. the example of a phone icon in https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html. However, there could also be emoticons that are as fine-grained as characters. So, i cannot draw an exact line between emoticons and ideograms. In general, it is best to aim for a contrast ratio of 4.5:1. 
Anyway, for the Image Profile presenting images, SC 1.4.11 should be referenced.

**Comment by palemieux** [comment]
> Anyway, for the Image Profile presenting images, SC 1.4.11 should be referenced.

I do not expect images to be presented since IMSC is for timed text content, not generic image content overlay.

**Comment by gzimmermann** [comment]
I am okay with that. 
What about the SCs 3.1.1 and 3.1.2 on language? Will you include them as well?

**Comment by nigelmegitt** [comment]
> What about the SCs 3.1.1 and 3.1.2 on language? Will you include them as well?

Yes, they are already included in #526 - it may be easier for you to see them by looking at the Diff link at the end of the [first comment](https://github.com/w3c/imsc/pull/526#issue-387544849) in that pull request.

**Comment by gzimmermann** [comment]
Okay, i see.  Perfect.
